plugins/org.yakindu.sct.ui.editor/src/org/yakindu/sct/ui/editor/propertysheets/ValidatingEMFDatabindingContext.java

```diff
@@ -39,6 +41,9 @@ public ValidatingEMFDatabindingContext(EObject context, Shell shell) {
 
 	protected IStatus isWriteable() {
 		IFile file = WorkspaceSynchronizer.getFile(context.eResource());
+		if (file == null || !file.isAccessible())
+			return new Status(IStatus.ERROR, DiagramActivator.PLUGIN_ID,
+					"File does not exist" + context.eResource().getURI());
```

Comment:
Missing whitespace? Sth like: "File does not exist: "

Reply:
Yes, fixed